pyproject.toml

```diff
@@ -27,7 +27,7 @@ SQLAlchemy = "1.3.18"
 wcwidth = "0.2.5"
 more-itertools = "^8.6.0"
 mysql-connector-python = "^8.0.20"
-doltcli = "0.1.2"
+doltcli = "^0.1.4"
```

Comment:
did we skip a version?

Reply:
I committed `1.0.3` forgetting that the release process bumped the version... seemed inconsequential to skip a patch increment.

Reply:
skipping 0.1.3 is fine, releasing 1.0.3 would be somewhat disruptive